src/components/bottom-widget.tsx

```diff
@@ -46,9 +53,23 @@ const BottomWidgetContainer = styled.div`
   ${media.portable`padding: 0;`}
 `;
 
+interface FilterAnimationControllerProps {
+  filter: TimeRangeFilter & {animationWindow?: string};
+  children?: ReactNode;
```

Comment:
you don't need `children` here when use the `React.FC` to define component as shown below